tests/data/collapse-sample/results/sample-default.txt

```diff
@@ -1,21 +1,17 @@
-Thread_15758535;libsystem_pthread`thread_start;libsystem_pthread`_pthread_start;libsystem_pthread`_pthread_body;rg`std::sys::unix::thread::Thread::new::thread_start;rg`std::sys_common::thread::start_thread;rg`<F as alloc::boxed::FnBox<A>>::call_box;rg`__rust_maybe_catch_panic;rg`std::panicking::try::do_call;rg`std::sys_common::backtrace::__rust_begin_short_backtrace;rg`ignore::walk::Worker::run;rg`rg::search_parallel::_{{closure}}::_{{closure}};rg`<rg::search::SearchWorker<W>>::search_impl;rg`grep_searcher::searcher::Searcher::search_path;rg`std::fs::OpenOptions::_open;rg`std::sys::unix::fs::File::open;rg`std::sys::unix::fs::File::open_c;rg`std::sys::unix::cvt_r;libsystem_kernel`__open 617
```

Comment:
What caused these changes?

Reply:
All of the other collapsers produce **sorted** output. `sample` on master wasn't doing that. So I changed it to produce sorted output. Thus, I also had to sort the test output. That's the only difference between this file and what it is on master.